holoviews/plotting/renderer.py

```diff
@@ -386,7 +419,7 @@ class needed to render it with the current renderer.
 
 
     @classmethod
-    def html_assets(cls, core=True, extras=True, backends=None):
+    def html_assets(cls, core=True, extras=True, backends=None, script=False):
```

Comment:
Is this the only place where we will have to worry about embedding JS into script tags?

Reply:
I think so.